Update teams-exploratory.md (#6430)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-exploratory.md

Users don't fit the criteria if they:

- Currently or previously had Teams from a paid, unpaid, or trial license

> - Are in a tenant that used/received at least one special COVID offer.This is the link to the list of tenantIds currently blocked by the signup service : https://dev.azure.com/microsoft/Universal%20Store/_git/BSX.Signup.Aloha/pullrequest/5282454?_a=files&path=%2Fsrc%2FCommon%2FDomainsBusinessLayer%2FDomainLists%2FTeamsBlockedTenantIds.csv

Reviewer: ```suggestion
- Are in a tenant that used/received at least one special COVID offer. The following link lists the tenant IDs currently blocked by the signup service: https://dev.azure.com/microsoft/Universal%20Store/_git/BSX.Signup.Aloha/pullrequest/5282454?_a=files&path=%2Fsrc%2FCommon%2FDomainsBusinessLayer%2FDomainLists%2FTeamsBlockedTenantIds.csv
```
